core/src/main/java/org/elasticsearch/action/admin/indices/analyze/AnalyzeRequest.java

```diff
@@ -114,7 +114,7 @@ public String field() {
 
     @Override
     public ActionRequestValidationException validate() {
-        ActionRequestValidationException validationException = super.validate();
```

Comment:
on a second thought (and reading further). I think we should change approach here. It's confusing now that this class partially forces an index and partially not. I suggest we make an index in this class consistently optional (mark as nullable, document what it means, remove the validation check). We should also add a utility validateNonNullIndex that does what the current validate does. All the subclasses that want this, can use it.